nixos/modules/virtualisation/qubes.nix

```diff
@@ -0,0 +1,34 @@
+{lib, pkgs, config, ...}:
+let
+  inherit (lib) mkEnableOption mkIf;
+  cfg = config.virtualisation.qubes;
+in {
+  # FIXME: Should it be qubes-dom0 or smth?
```

Comment:
Let's mirror the Xen module. dom0 stuff is at the top-level (`virtualisation.qubes.*`) and domU stuff is under `virtualisation.qubes.guest.*`.

The file itself should be `qubes-dom0.nix` though.